default_data/settings.gd

```diff
@@ -7,27 +7,27 @@ const data :="""
 [user]
 fps_overlay=false
 sensitivity=1.0
-sensitivity_game="Counter Strike 2"
+sensitivity_game=0
```

Comment:
Are you sure it's good idea to store this as an int?
Strings make it more user-friendly and readable. Plus, if you ever modify the order of the list, ints will ruin settings for all existing users.

Reply:
I also like the string, but, I those "default settings" should be exactly the same as the file generated in the `user` folder. And that file in `user` folder was been saved as a 0.

Changing this should be it's own pull request after this one is merged.